spec/action_mailer_courier_transformer_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rspec"
+require "courier_rails/action_mailer_courier_transformer"
+
+def mockMailObject(obj)
+  require "json"
```

Comment:
These "require" calls might as well go at the top with the others; not sure I've seen them in a method before